icx-http-server/Cargo.toml

```
@@ -0,0 +1,20 @@
+[package]
+name = "icx-http-server"
+version = "0.1.0"
+authors = ["DFINITY Stiftung <sdk@dfinity.org>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+candid = "0.6.15"
+clap = "3.0.0-beta.2"
+delay = "0.3.1"
+hyper = { version = "0.14.4", features = ["full"] }
+ic-agent = { path = "../ic-agent" }
+ic-types = { path = "../ic-types" }
+ic-utils = { path = "../ic-utils" }
+tokio = { version = "1", features = ["full"] }
```

I wonder if we ought to specify the full version, so that we have to be explicit about getting new tokio versions:
```suggestion
tokio = { version = "1.2.0", features = ["full"] }
```